plugins/plugin-debugger/che-plugin-debugger-ide/src/main/java/org/eclipse/che/plugin/debugger/ide/debug/AbstractDebugger.java

```diff
@@ -444,6 +443,7 @@ protected void startDebugger(final DebugSessionDto debugSessionDto) {
             LocationDto locationDto = dtoFactory.createDto(LocationDto.class);
             locationDto.setLineNumber(b.getLineNumber() + 1);
             locationDto.setResourcePath(b.getPath());
+            locationDto.setResourceProjectPath(b.getFile().getProject().getProjectConfig().getPath());
```

Comment:
mb it would be better
```
dtoFactory.createDto(LocationDto.class);
                 .withLineNumber(b.getLineNumber() + 1);
                 .withResourcePath(b.getPath());
                 .withResourceProjectPath(b.getFile().getProject().getProjectConfig().getPath());
```